Update getting-started.md (#76)
hashicorp/consul-api-gateway: dev/docs/getting-started.md

```bash
kubectl delete -f dev/config/k8s/consul-api-gateway.yaml
```

> ## Deploying a custom Docker image

Reviewer: ```suggestion
## Deploying a custom Docker image on a development cluster
```

- Create a Docker image from your local branch with `make docker`

> - Upload image to k8s cluster, for a local kind cluster the command is [`kind load docker-image`](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster)

Reviewer: ```suggestion
- *Optional*: Some k8s setups require loading a custom image into the runtime environment for it to get pulled by worker nodes. For a local kind cluster you can load the image using the subcommand [`kind load docker-image`](https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster)
```

> - In `config/deployment/deployment.yaml`, edit `image` to point to the name of the image you uploaded to the cluster.

Author: ```suggestion
- In [`config/deployment/deployment.yaml`](https://github.com/hashicorp/consul-api-gateway/blob/main/config/deployment/deployment.yaml#L23), edit `image` to point to the name of the image you uploaded to the cluster.
```
